services/util/test/TestFlashSpi.cpp

```diff
@@ -42,6 +42,15 @@ TEST_F(FlashSpiTest, Construction)
     EXPECT_EQ(4096, flash.SizeOfSector(0));
 }
 
+TEST_F(FlashSpiTest, ConstructionWithConfig)
+{
+    services::FlashSpi::Config config{ .nrOfSubSectors = 1024, .sizeSector = 8192, .sizeSubSector = 1024, .sizePage = 64 };
+    services::FlashSpi flashConfigured{ spiMock, config };
+
+    EXPECT_EQ(1024, flashConfigured.NumberOfSectors());
```

Comment:
If you're testing whether the variables have been correctly initialized, please use different values for the different parameters